### PR TITLE
mpd 0.19.10

### PR DESCRIPTION
**Pull request by michaelpq**
This is a bugfix release:
* input
 - curl: fix deadlock on small responses
 - smbclient: fix DFF playback
* decoder
 - ffmpeg: improve seeking accuracy
 - fix stuck stream tags
* encoder
 - opus: fix bogus granulepos
* output
 - fix failure to open device right after booting
* neighbor
 - nfs: fix deadlock when connecting
* fix "single" mode breakage due to queue edits

### CODE REVIEW

File: Library/Formula/mpd.rb [hunk 1]
```diff
@@ -4,13 +4,12 @@ class Mpd < Formula
...
   stable do
-    url "http://www.musicpd.org/download/mpd/0.19/mpd-0.19.9.tar.xz"
-    sha256 "47851423053cd38cfad65be5985b41b7cd5bdbe8d0d13378e11748a28b19f26f"
+    url "http://www.musicpd.org/download/mpd/0.19/mpd-0.19.10.tar.xz"
+    sha256 "c386eb3d22f98dc993b5ae3c272f969aa7763713483c6800040ebf1791b15851"
   end
 
   bottle do
     cellar :any
-    revision 1
```
Comment (bfontaine) [inline Library/Formula/mpd.rb]: Please don’t change the bottle block, it’ll be handled automatically by our bot. Thanks!

Comment (michaelpq) [inline Library/Formula/mpd.rb]: No problem. Updated.

**Comment by DomT4** [comment]
Please remove the `revision 1` on line 4. Revisions need to be reset with each `stable` update.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/367293a607169bd418d44745413a7f9d24222f3c. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**